src/components/patterns/link.tsx

```diff
@@ -44,10 +44,14 @@ export const DrupalActionLink = ({href, children, ...props}: PropsWithChildren<{
     <Link
       href={href}
       {...props}
-      className={twMerge("relative pr-30 hocus:text-brick no-underline rs-mt-neg1 pt-10", props.className)}
+      className={'pt-10'}
     >
-      {children}
-      <ChevronRightIcon className="inline absolute top-0 right-0 h-full pt-10"/>
+      <tspan  className={twMerge("relative pr-30 no-underline rs-mt-neg1 hover:bg-black-10 hover:text-brick hocus:underline focus:bg-black-true active:bg-black-true", props.className)}
```

Comment:
whats `tspan`

Reply:
I was curious about that myself. PHPStorm suggested it, so I went with it to find out. I forgot to push the version with `span`.

Reply:
Seems to be a tag specific to SVGs: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/tspan

Reply:
ahh. interesting. I've done very little SVG coding.